enterprise_access/apps/api/serializers/subsidy_requests.py

```diff
@@ -121,3 +123,34 @@ def update(self, instance, validated_data):
         # Pop enterprise_customer_uuid so that it's read-only for updates.
         validated_data.pop('enterprise_customer_uuid', None)
         return super().update(instance, validated_data)
+
+
+class LearnerCreditRequestConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the `LearnerCreditRequestConfiguration` model.
+    """
+
+    class Meta:
+        model = LearnerCreditRequestConfiguration
+        fields = "__all__"
+        read_only_fields = ["uuid", "active", "created", "modified"]
+
+
+class LearnerCreditRequestSerializer(SubsidyRequestSerializer):
+    """
+    Serializer for the `LearnerCreditRequest` model.
+    """
+
+    learner_credit_request_config = serializers.PrimaryKeyRelatedField(
+        queryset=LearnerCreditRequestConfiguration.objects.all(),
+        required=False,
+        allow_null=True,
+    )
```

Comment:
Question: Curious why we added this field, given that we'll only be getting policy uuid in the request.